include/unicore-mx/usbd/usbd.h

```diff
@@ -135,6 +135,16 @@ enum usbd_speed {
 
 typedef enum usbd_speed usbd_speed;
 
+	/** Optional features */
+enum usbd_backend_features{
+		USBD_FEATURE_NONE = 0,
+		USBD_PHY_EXT = (1 << 0),
+		USBD_VBUS_SENSE = (1 << 1),
+		USBD_VBUS_EXT = (1 << 2)
+		//* provide usb-core auto power-up/down on connect/disconnect
```

Comment:
"//*" (IMO) Please C style comment only (code consistancy).
@danielinux @brabo what do you think?